src/OrchardCore.Modules/OrchardCore.Widgets/Views/ContentCard.Edit.cshtml

```diff
@@ -0,0 +1,3 @@
+@*Alternate this shape using in Module - like ContentCard-Flow.Edit.cshtml *@
+@**TODO: Provide default Shape here*@
+
```

Comment:
Do we need this file? is it just an alternate, that could be implemented? 
just trying to understand why it's empty?

Reply:
ContentCard.Edit.cshtml and ContentCard.Fields.Edit.cshtml are base shapes with empty output - these shapes are overridden by alternates in Flow module.   ASP.NET doesn't like removing them.

`System.Exception: Shape type 'ContentCard_Edit' not found`